src/typescript-service.ts

```diff
@@ -2,7 +2,7 @@ import { Operation } from 'fast-json-patch'
 import iterate from 'iterare'
 import { castArray, merge, omit } from 'lodash'
 import { toPairs } from 'lodash'
-import hashObject = require('object-hash')
+import * as hashObject from 'object-hash'
```

Comment:
Using `* as` for functions that are exported with `module.exports` is not semantically correct - according to the ES6 spec `hashObject` would be a frozen module record object, not a function.

The right solution for this would be to use the new `esModuleInterop` flag and import the default import.